src/parity_testing/auth/testSuites/resetPassword/Test_that_Cognito_is_called_with_given_payload_and_returns_successful_data.json

```diff
@@ -1,29 +0,0 @@
-{
```

Comment:
This is already tested in `AuthResetPasswordResult_object_is_returned_when_reset_password_succeeds.json`